assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -19,42 +18,34 @@ import EmptyCart from './empty-cart';
  * Component to handle edit mode of "Cart Block".
  */
 const CartEditor = ( { className } ) => {
-	const [ isFullCartMode, setFullCartMode ] = useState( true );
-
-	const toggleFullCartMode = () => {
-		setFullCartMode( ! isFullCartMode );
-	};
-
-	const getBlockControls = () => {
-		return (
-			<BlockControls className="wc-block-cart-toolbar">
-				<Toolbar>
-					<TextToolbarButton
-						onClick={ toggleFullCartMode }
-						isToggled={ isFullCartMode }
-					>
-						{ __( 'Full Cart', 'woo-gutenberg-products-block' ) }
-					</TextToolbarButton>
-					<TextToolbarButton
-						onClick={ toggleFullCartMode }
-						isToggled={ ! isFullCartMode }
-					>
-						{ __( 'Empty Cart', 'woo-gutenberg-products-block' ) }
-					</TextToolbarButton>
-				</Toolbar>
-			</BlockControls>
-		);
-	};
+	const [ currentView, setCurrentView ] = useState( 'full' );
 
 	return (
 		<div className={ className }>
-			{ getBlockControls() }
-			{ isFullCartMode && (
+			<ViewSwitchControl
+				label={ __( 'Edit', 'woo-gutenberg-products-block' ) }
+				onChange={ ( view ) => setCurrentView( view ) }
+				views={ [
+					{
+						value: 'full',
+						name: __( 'Full Cart', 'woo-gutenberg-products-block' ),
+					},
+					{
+						value: 'empty',
+						name: __(
+							'Empty Cart',
+							'woo-gutenberg-products-block'
+						),
+					},
+				] }
+				selected={ currentView }
```

Comment:
One thing I wish we can do is to directly provide the components (views) via a render prop or to pass it to the component, I feel like it would make more sense.